spec/models/address_spec.rb

```diff
@@ -71,18 +71,36 @@
     end
 
     context 'when postcode is present' do
-      let(:params) { { postcode: 'AA1 2BB' } }
+      context 'and is a valid UK postcode' do
+        let(:params) { { postcode: 'AA1 2BB' } }
 
-      it 'raises a validation around geocoding' do
-        expect(subject.errors.full_messages).to include('Point geocoder lookup failed')
+        it 'raises a validation around geocoding' do
+          expect(subject.errors.full_messages).to include('Point geocoder lookup from postcode failed')
```

Comment:
This might be useful: https://github.com/guidance-guarantee-programme/pension_guidance/blob/master/spec/controllers/locations_controller_spec.rb